src/components/PropertyActions.tsx

```diff
@@ -0,0 +1,25 @@
+type PropertyActionsProps = {
+  onLike: () => Promise<void>;
+  onDislike: () => Promise<void>;
+};
+
+function PropertyActions({ onLike, onDislike }: PropertyActionsProps) {
+  return (
+    <div className="flex flex-row justify-center align-center">
+      <button
+        onClick={onLike}
+        className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded align-right m-2"
+      >
+        Like
+      </button>
+      <button
+        onClick={onDislike}
+        className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded align-right m-2"
+      >
+        Dislike
+      </button>
+    </div>
```

Comment:
Its  a good idea to keep components in its own folder. You can undo this change.